### ISSUE TITLE
No Render-Help Reviewing the Code

### ISSUE THREAD

**Issue by FabrizioFioretti** [stolksdorf/homebrewery#785]
## No render on the right, just a page count
After saving my brew, the render disappears, and the pdf appears messed up with out of place background and wrong formatting on same pages.
I know this has been reported already, and I know it's probably because something in the code is out of place, but this is my first time using GitHub/The Homebrewery/coding in general.
I think I already noticed and removed an out of place column break, but there are obviously other errors or issues I cannot find.
Can someone help me locate the errors I am missing, and advise me on how to avoid them?

Thank you very much!

**Brew code to reproduce** : <details><summary>Click to expand</summary><code><pre>

https://homebrewery.naturalcrit.com/edit/H1gO-xj-2m

</pre></code></details>

**Comment by FabrizioFioretti** [comment]
I made a few corrections and it seems to work now, but I am still afraid I missed something critical, and that maybe I could have this problem again.
This stems mainly from the fact that I am not sure I know what exactly was wrong.
Corrections I made:
Adding page numbering code in a couple pages missing.
Adding a column break in the first page (lines 29 to 35)
Adding multiple > to spacing in notes.
Removing empty lines at the end of descriptive text boxes before the </div>
Removed some unnecessary lines.

The "Natalja" monster stat box behaved a little weirdly, as it did seem to morph into a large stat block even though it did not have the two lines of ____ over it but just one.

https://homebrewery.naturalcrit.com/edit/H1gO-xj-2m
